package.json

```
@@ -53,7 +53,7 @@
     "rxjs": "6.x"
   },
   "devDependencies": {
-    "@openmrs/esm-framework": "next",
+    "@openmrs/esm-framework": "^5.7.3-pre.2169",
```

```suggestion
    "@openmrs/esm-framework": "next",
```

This should not be changed. (same for line 88 )